calico-cloud/users/create-custom-role-for-entra-id-group.mdx

```diff
@@ -0,0 +1,55 @@
+---
+description: Create custom roles for Entra ID groups.
+---
+
+import IconUser from '/img/icons/user-icon.svg';
+
+# Create a custom role for an Entra ID group
+
+If you have Microsoft Entra ID configured as your identity provider, you can assign custom roles to Entra ID (formerly Azure AD) security group.
+This lets you manage role-based access to Calico Cloud directly from your identity provider portal.
+
+## Prerequisites
+
+* You have owner or administrator permissions to the Calico Cloud Manager UI.
+* You have administrator permissions for your organization in the Azure Portal.
+
+## Create and configure an Azure web app
```

Comment:
If `/users/user-management#add-your-own-identity-provider` is not being removed,  the section "Create and configure an Azure web app" is really part of the `#add-your-own-identity-provider` prerequisite so should not be repeated here